data/migrations/20200601193619_event_modifiers.js

```diff
@@ -7,6 +7,7 @@ exports.up = function(knex) {
             tbl.text('instructions').notNullable();
             tbl.decimal('servings', 5, 2).notNullable();
             tbl.binary('photo');
+            tbl.string('source', 64);
```

Comment:
@Aaroneld I made some additional changes to the table schema. Please revise your dbDesigner.net schema documentation accordingly.